person.rb

```diff
@@ -20,6 +21,10 @@ def validate_name
     @name = @corrector.correct_name(name)
   end
 
+  def add_rental(date, person)
+    Rental.new(date, self, person)
+  end
```

Comment:
- [ ]  This method for adding rentals should take book and date as parameters, then the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. See the code snippet below for clarifications and kindly fix that.

```ruby
def add_rental(book, date)
  Rental.new(date, book, self)
end

Reply:
This change has been implemented well